manifests/t/ThielickeITSolutions/TIPP10/2.1.0/ThielickeITSolutions.TIPP10.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: ThielickeITSolutions.TIPP10
+PackageVersion: 2.1.0
+PackageLocale: en-US
+Publisher: (c) 2006-2011, Tom Thielicke IT Solutions
+PackageName: TIPP10
+License: (C) 2022 Thielicke IT Solutions

```

Comment:
A copyright isn't a license; Please update this with actual license terms such as Proprietary, MIT, GPL-2.0, or other terms as appropriate